### PR TITLE
[hbase10] Checkstyle updates for HBase 1.0 binding.

### PR DESCRIPTION
**Pull request by allanbank**

### CODE REVIEW

File: hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java [hunk 1]
```diff
@@ -59,460 +58,440 @@
...
-	  {
-	      final TableName tableName = TableName.valueOf(table);
-	      HTableDescriptor dsc = _connection.getTable(tableName).getTableDescriptor();
-	  }
-      catch (IOException e)
-	  {
-	      throw new DBException(e);
-	  }
+public class HBaseClient10 extends com.yahoo.ycsb.DB {
+  private static final int HTTP_ERROR = -2;
```
Comment (cmccoy) [inline hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java]: This isn't used, as far as I can tell.

File: hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java [hunk 2]
```diff
@@ -59,460 +58,440 @@
...
+
+  private Configuration config = HBaseConfiguration.create();
+
+  private boolean debug = false;
+
+  private String tableName = "";
+  private Connection connection = null;
+
+  // Depending on the value of _clientBuffering, either _bufferedMutator
+  // (_clientBuffering) or _hTable (!_clientBuffering) will be used.
```
Comment (cmccoy) [inline hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java]: These comments refer to old variable names.

**Comment by cmccoy** [comment]
Thanks, @allanbank. Looks good to me. A few little things above; I'm happy to fix them if you don't have time.

**Comment by allanbank** [comment]
Updated the commit based on feedback.  Thanks @cmccoy.

**Comment by cmccoy** [comment]
+1, thanks.

**Pull request merged**